apps/costcontrol/test/unit/startup_test.js

```diff
@@ -427,4 +427,73 @@ suite('Application Startup Modes Test Suite >', function() {
       CostControlApp.init();
     }
   );
+
+  test(
+    'DSDS Ensure the FTE will be closed the second time when there are a data' +
+      ' slot change',
+    function(done) {
+      var defaultDataSlotId = 0, newDataSlotId = 1,
+          dataSlot = 'ril.data.defaultServiceId';
+      var applicationMode = 'DATA_USAGE_ONLY';
+      function setupSims() {
+        MockNavigatorSettings.mSettings[dataSlot] = defaultDataSlotId;
+
+        setupCardState({cardState: 'ready'});
+        window.Common = new MockCommon({ });
+        window.ConfigManager = new MockConfigManager({
+          fakeSettings: { fte: true },
+          applicationMode: applicationMode
+        });
+
+        MockNavigatorMozMobileConnections.mAddMobileConnection(
```

Comment:
nit: reformat this to be
```javascript
MockNavigatorMozMobileConnections.mAddMobileConnection(
  { iccId: '0000000' },
  1
);
```
or

MockNavigatorMozMobileConnections
  .mAddMobileConnection({ iccId: '0000000' }, 1);